proxies/live/apiproxy/targets/sync-wrap.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Don't know if it makes a difference, but do we need these xml tags?

Reply:
No strictly speaking - they are inferred if you leave them out